src/libraries/Microsoft.Bcl.Memory/src/Microsoft.Bcl.Memory.csproj

```diff
@@ -0,0 +1,36 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;$(NetFrameworkMinimum);$(NetCoreAppCurrent)</TargetFrameworks>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <UseCompilerGeneratedDocXmlFile>false</UseCompilerGeneratedDocXmlFile>
+    <IsPackable>true</IsPackable>
+    <!-- Disabling baseline validation since this is a brand new package.
+         Once this package has shipped a stable version, the following line
+         should be removed in order to re-enable validation. -->
+    <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
+    <PackageDescription>
+      Provides Base64Url encoding, decoding and validation APIs support for .NET Framework and .NET Standard.
+
+      Commonly Used Types:
+      System.Buffers.Text.Base64Url
+    </PackageDescription>
+  </PropertyGroup>
+
+  <!-- DesignTimeBuild requires all the TargetFramework Derived Properties to not be present in the first property group. -->
+  <PropertyGroup>
+    <IsPartialFacadeAssembly Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETCoreApp'">true</IsPartialFacadeAssembly>
```

Comment:
I _think_ we should just include the "..ref/Microsoft.Bcl.Memory.Forwards.cs" file here instead of setting this property. @ericstj opinions?

Reply:
Correct, it's better to hand author the forwards here.  No need to use partial facade generation.